<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainingNewModelDialog</class>
 <widget class="QDialog" name="TrainingNewModelDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="windowTitle">
   <string>New Model</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="introLabel">
     <property name="text">
      <string>Configure a training run and optional base model.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="configurationGroup">
     <property name="title">
      <string>Training Parameters</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>Model name</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nameLineEdit">
        <property name="placeholderText">
         <string>strategy_ft</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="baseLabel">
        <property name="text">
         <string>Base model</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="baseComboBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="deviceLabel">
        <property name="text">
         <string>Device</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="deviceComboBox"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="epochsLabel">
        <property name="text">
         <string>Epochs</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="epochsSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
        <property name="value">
         <number>40</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="learningRateLabel">
        <property name="text">
         <string>Learning rate</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="learningRateSpinBox">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>0.000001</double>
        </property>
        <property name="maximum">
         <double>0.100000</double>
        </property>
        <property name="singleStep">
         <double>0.000500</double>
        </property>
        <property name="value">
         <double>0.002000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="datasetLabel">
        <property name="text">
         <string>Dataset (optional)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QWidget" name="datasetWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QLineEdit" name="datasetLineEdit">
           <property name="placeholderText">
            <string>Use prepared dataset folder</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="browseDatasetButton">
           <property name="text">
            <string>...</string>
           </property>
           <property name="toolTip">
            <string>Select dataset directory</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="featuresLabel">
        <property name="text">
         <string>Features</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="v2FeaturesCheckBox">
        <property name="text">
         <string>Use geometry-aware feature set (v2)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TrainingNewModelDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TrainingNewModelDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>496</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
